client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/index.tsx

```diff
@@ -341,15 +341,26 @@ const designSetup: Step = function DesignSetup( { navigation, flow } ) {
 		window.scrollTo( { top: 0 } );
 	}, [ isForceStaticDesigns ] );
 
-	if ( selectedDesign && isPreviewingDesign && ! showGeneratedDesigns ) {
-		const isBlankCanvas = isBlankCanvasDesign( selectedDesign );
-		const designTitle = isBlankCanvas ? translate( 'Blank Canvas' ) : selectedDesign.title;
-		const shouldUpgrade = selectedDesign.is_premium && ! isPremiumThemeAvailable;
-		const previewUrl = getDesignPreviewUrl( selectedDesign, {
+	useEffect( () => {
+		if ( showGeneratedDesigns ) {
+			window.scrollTo( { top: 0 } );
+		}
+	}, [ isPreviewingDesign, showGeneratedDesigns ] );
```

Comment:
I think that for the traditional design picker, we want to keep the scroll position so that they can see the design they've selected (for previewing). We can still merge the two `useEffect` though!